lms/djangoapps/ccx/views.py

```diff
@@ -179,6 +179,11 @@ def dashboard(request, course, ccx=None):
                 model='CourseEnrollment',
             )
         )
+        # if course licensing is enabled, pending enrollments will be in the context.
+        context['pending_ccx_members'] = run_extension_point(
+            'PCO_GET_PENDING_ENROLLMENTS_FOR_CCX',
+            ccx_id=ccx_locator,
+        ) if run_extension_point('PCO_ENABLE_COURSE_LICENSING') else ()
```

Comment:
Why is "()" the default in case "PCO_GET_PENDING_ENROLLMENTS_FOR_CCX" returns False? @anfbermudezme